README.md

```diff
@@ -97,8 +97,10 @@ message Product {
 
 Results in the following JSON Schema files:
 
-- `*.schema.json` files are generated with underscore-case fields
-- `*.jsonschema.json` files are generated with camelCase
+- `*.schema.json` files are generated with the protobuf field name (e.g. `product_id`, `product_name`)
```

Comment:
Should we mention these are generated with `names=all` (default) or `names=proto`?